superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -86,6 +86,11 @@ struct BenchArgs {
     // Whether check data after copy.
```

Comment:
We don't need P2P anymore?

Reply:
In existing superbench code, dtod covers both local GPU memory bandwidth and p2p. In earlier version of this PR, we wan t to separate into dtod and ptop and use option to whether to enable fine grained memory. Now it’s directly controlled according to whether source GPU ID == destination GPU ID. So no ptop required anymore.